app/src/main/res/layout-land/ongoing_story_card.xml

```diff
@@ -53,37 +53,33 @@
         app:layout_constraintTop_toBottomOf="@id/lesson_thumbnail">
 
         <TextView
+          style="@style/Subtitle2"
```

Comment:
This style is incorrect because this style refers to `14sp` textSize but this view needs `16sp`.